tests/app/data/test_schema_validation.py

```diff
@@ -148,6 +164,18 @@ def _parse_id_values(self, schema_json, ignored_keys):
                     if isinstance(schema_item, dict):
                         yield from self._parse_id_values(schema_item, ignored_keys)
 
+    def _parse_alias_values(self, schema_json, ignored_keys):
+        for k, v in schema_json.items():
+            if k == 'alias':
+                yield v
+            elif k in ignored_keys:
```

Comment:
The function needs a name change or the content needs reducing. If its purely dealing with alias' why are we worrying about 171 down.